$NetBSD: patch-ae,v 1.8 2016/04/15 08:47:50 wiz Exp $

 - Don't assert GNU ld on Solaris, where it might not be true.

--- include/local.h.orig	2016-02-29 14:44:47.000000000 +0000
+++ include/local.h
@@ -253,7 +253,9 @@ extern snd_lib_error_handler_t snd_err_m
 
 /*
  */
+#ifndef __sun
 #define HAVE_GNU_LD
+#endif
 #define HAVE_ELF
 #define HAVE_ASM_PREVIOUS_DIRECTIVE
 
